apps/breethe/test/breethe/sources/eea/csv_test.exs

```diff
@@ -0,0 +1,38 @@
+defmodule Breethe.Sources.EEA.CSVTest do
+  use Breethe.DataCase
+
+  alias Breethe.Sources.EEA.CSV
+
+  @sample_invalid_data "network_countrycode,network_localid,network_name,network_namespace,network_timezone,pollutant,samplingpoint_localid,samplingpoint_namespace,samplingpoint_x,samplingpoint_y,coordsys,station_code,station_localid,station_name,station_namespace,value_datetime_begin,value_datetime_end,value_datetime_inserted,value_datetime_updated,value_numeric,value_validity,value_verification,station_altitude,value_unit\r\nBG,NET-BG001A,National air network,BG.BG-ExEA.AQ,http://dd.eionet.europa.eu/vocabulary/aq/timezone/UTC+02,SO2,SPO-BG0071A_00001_100,BG.BG-ExEA.AQ,27.720956,42.65975799999999,EPSG:4979,BG0071A,STA-BG0071A,Nesebar,BG.BG-ExEA.AQ,2019-11-10 23:00:00+01:00,2019-11-11 00:00:00+01:00,2019-11-11 01:29:21+01:00,+01:00,,-1,1,27,ug/m3\r\n"
+
+  @sample_valid_data "network_countrycode,network_localid,network_name,network_namespace,network_timezone,pollutant,samplingpoint_localid,samplingpoint_namespace,samplingpoint_x,samplingpoint_y,coordsys,station_code,station_localid,station_name,station_namespace,value_datetime_begin,value_datetime_end,value_datetime_inserted,value_datetime_updated,value_numeric,value_validity,value_verification,station_altitude,value_unit\r\nGB,NET-GB001A,National air network,GB.GB-ExEA.AQ,http://dd.eionet.europa.eu/vocabulary/aq/timezone/UTC+02,SO2,SPO-BG0071A_00001_100,GB.GB-ExEA.AQ,00.72,34.65975799999999,EPSG:4979,GB0071A,STA-GB0071A,London,GB.GB-ExEA.AQ,2019-11-10 23:00:00+01:00,2019-11-11 00:00:00+01:00,2019-11-11 01:29:21+01:00,+01:00,3,-1,1,27,ug/m3\r\n"
```

Comment:
Might be easier to have these in fixture files? It's a bit hard to follow here what the actual data is in a super long string like this.

Reply:
Wouldn't that just move the problem from one file to another though?

Reply:
Perhaps programmatically creating the data would be a better solution?

Reply:
Hm, I think actually the best solutions would be keeping it here but properly formatting it as a multiline string: https://elixir-examples.github.io/examples/multiline-strings-heredocs

Reply:
fixed - this is the simplest I could do without implementing a custom sigil